surl/__init__.py

```diff
@@ -239,33 +224,56 @@ def get_config_from_cli(parser, auth_dir):
         except ConfigError:
             raise CliError(
                 "** Deprecated or Broken authentication file, "
-                "please delete it and login again:\n  $ rm {}".format(
-                    auth_path
-                )
+                "please delete it and login again:\n  $ rm {}".format(auth_path)
             )
 
         return config, remainder
 
+    # TODO: change this to use either snap or charm based on URL
+    packages = (
+        [endpoints.Package(name, "snap") for name in args.snaps] if args.snaps else []
+    )
+    # NOTE: this will go away when we transition to click
```

Comment:
I have a personal bugbear about needless "NOTE:" prefixes. Doesn't the comment have identical meaning without it? But obviously this is beyond trivial - do what you prefer.

Reply:
I left if there mostly so I could look back at what I needed to change, although looking at it now the entire comment seems irrelevant. I'm leaving it in though it'll disappear organically soonish.